Sandbox/src/Game/Interface/GearInterface.cpp

```diff
@@ -1,17 +1,83 @@
 #include "GearInterface.h"
 
 GearInterface::GearInterface(Gear& gear)
-	: IMenuTab("Graphics/panel_brown.png", 32, false, false), mGear(gear)
+	: IMenuTab("Graphics/panel_gear.png", 32, false, false), mGear(gear)
 {
-	for (int y = 0; y < 5; y++)
+	InitializeGearPositions();
+}
+
+void GearInterface::InitializeGearPositions()
+{
+	mGearPositions = {
+	{ Gear::EquipmentSlot::head, {COL_THREE_X, ROW_ONE_Y} },
+	{ Gear::EquipmentSlot::cape, {COL_TWO_X, ROW_TWO_Y} },
+	{ Gear::EquipmentSlot::neck, {COL_THREE_X, ROW_TWO_Y} },
+	{ Gear::EquipmentSlot::weapon, {COL_ONE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::chest, {COL_THREE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::shield, {COL_FIVE_X, ROW_THREE_Y} },
+	{ Gear::EquipmentSlot::legs, {COL_THREE_X, ROW_FOUR_Y} },
+	{ Gear::EquipmentSlot::gloves, {COL_ONE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::boots, {COL_THREE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::ring, {COL_FIVE_X, ROW_FIVE_Y} },
+	{ Gear::EquipmentSlot::ammo, {COL_FOUR_X, ROW_TWO_Y} },
+	};
+
+	mGearDefaultSprites = {
```

Comment:
Create an std::vector<Square::Sprite*> mIcons in IMenuTab, then use this class to add your sprite icons to it. If we create a new container in the derived classes we would need to create a new render method which adds a lot of unnecessary complexity.